controllers/paymentController.php

```diff
@@ -55,6 +60,7 @@ public function __construct()
         $this->id = WcPagantisGateway::METHOD_ID;
         $this->has_fields = true;
         $this->method_title = ucfirst($this->id);
+        $this->paymentProcessingToken = strtoupper(md5(uniqid(rand(), true)));
```

Comment:
Please change it, we told you twice:  
- https://pagantis.slack.com/archives/G017X8BB0C8/p1600860303011700
- https://pagantis.slack.com/archives/G017X8BB0C8/p1600854426008600

Reply:
if the name is identical in both payment methods it create issues so I have renamed it to ` $urlToken` for the normal method and ` $urlToken4x` for the 4x method